### ISSUE TITLE
C# Syntax Highlight: Null Propagation Operator

### ISSUE THREAD

**Issue by alexr00** [dotnet/csharp-tmLanguage#163]
Originally from @wwdenis here: https://github.com/microsoft/vscode/issues/79872

## Details

- VSCode Version: 1.37.1
- OS Version: Windows_NT x64 10.0.18362

Steps to Reproduce:

![CSHARP_HIGHLIGHT](https://user-images.githubusercontent.com/21691662/63760049-e7cfaf80-c8be-11e9-98c8-1e853e7075ac.gif)

Snippet:
```cs
        var text = nullable?
            .ToString();
        int x = 0;
```
If you move the `.ToString();` on to the same line as the `?` then the following code gets the expected scopes.

**Comment by Muchiachio** [comment]
Looks like a dup of #107.

**Comment by mehdi-ichka** [comment]
It has been almost 3 years now. the same issue is still on version 1.54.1
Is there any updates plz ?

**Comment by wise0704** [comment]
Fixed in #271

**Comment by wwdenis** [comment]
Thanks everyone!
